packages/load-script/src/dom-operations.js

```diff
@@ -3,14 +3,20 @@ import { handleRequestError, handleRequestSuccess } from './callback-handler';
 
 const debug = debugFactory( 'lib/load-script/dom-operations' );
 
-export function createScriptElement( url ) {
+export function createScriptElement( url, args ) {
```

Comment:
This looks like a worthy change that would need to be documented as part of the package's README.